12_glob_lab/globlab.txt

```diff
@@ -1,29 +1,59 @@
 Glob Lab.
 https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
-1. Type in a glob pattern to match:
+1. 
+
+Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+ 
+ 1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+ 
+ls -d [[:lower:]]*/*
+ 2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+ 
+ls -d [[:upper:]]*/* - for some reason shows the following:
+Doc/README.txt Doc/test.c
+Why test.c as well?
+
+ 3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
-  4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
+ 
+echo *[a,e,i,o,u]
+
+ 4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
 
-  5. All files with extension ".c" that are in a folder named "src"
+ 
+echo *[!aeiou]
+
+ 5. All files with extension ".c" that are in a folder named "src"
   Answer:
 
-  6. All files with extension ".c" under any folder
+
+
+ls *.c src/
+  
+6. All files with extension ".c" under any folder
   Answer:
 
-2. Write a command that would delete all files ending with a ".png" under any subdirectory.
+
+find -name '*.c'
+
+     2. Write a command that would delete all files ending with a ".png" under any subdirectory
+find -name '*.png* -delete
+
+     
 
-3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
+3. The command "ls -l" can take the name of a file or directory and show details on them. From within "wildcards" folder run "ls -l [A-Z]*". What was printed? Why?
+Printed same thing as in answer to question 2, I don't understand why only these files are printed 
+

```

Comment:
Not sure I understand - what command are you running and what output do you expect to see?